zookeeper-api/src/main/java/org/apache/helix/zookeeper/constant/ZkSystemPropertyKeys.java

```diff
@@ -36,6 +36,16 @@
   public static final String ZK_SERIALIZER_ZNRECORD_AUTO_COMPRESS_ENABLED =
       "zk.serializer.znrecord.auto-compress.enabled";
 
+  /**
+   * Setting this property to true enables ZNode bucketization, an operation to divide a ZNRecord
+   * into specified buckets. This property applies to
+   * {@link org.apache.helix.zookeeper.datamodel.ZNRecordBucketizer}.
+   * <p>
+   * The default value is "true" (enabled).
+   */
+  public static final String ZK_BUCKETIZE_ZNRECORD_ENABLED =
+      "zk.bucketize.znrecord.enabled";
```

Comment:
How about `zk.znrecord.bucketize.enabled` or `zk.znrecord.bucketize.enabled`(which is consistent with the class `ZNRecordBucketizer`)?

Reply:
TFTR. Good suggestion. Will update.